playground/nextjs-app-router/package.json

```diff
@@ -17,7 +17,6 @@
     "watch:tailwind": "mkdir -p ./node_modules/@coinbase/onchainkit/src && nodemon --watch ../../src/styles.css --exec 'cp ../../src/styles.css ./node_modules/@coinbase/onchainkit/src/styles.css'"
   },
   "dependencies": {
-    "@coinbase/onchainkit": "^0.35.0",
```

Comment:
why are we removing this ?

Reply:
We aren't, this is just for my local dev